network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -127,6 +127,7 @@ def next_page(self):
         Only applies to Chapter Publication (sub-Publication Pages).
         Returns a Page object or None.
         """
+        next_page = None
```

Comment:
We probably want to set this to `self.get_parent()` instead of `None`, so that even if `self.is_chapter_page` is false, we return something with a `.url` property (otherwise `<None>url` in the template might still fail).

The following block logic might need an update though, along the lines of:
```python
next_page = self.get_parent()
if self.is_chapter_page
    sibling = self.get_siblings......
    if sibling:
        next_page = sibling
return next_page
```
```